packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js

```diff
@@ -81,42 +88,45 @@ const WatchlistCard = props => {
 
   const onDragEnd = e => {
     if (e.dataTransfer.dropEffect === 'none') {
-      props.bindings.onDropOutside(props.symbol, stockData.name);
+      bindings.onDropOutside(symbol, stockData.name);
     }
     setDragging({ isDragging: false });
   };
 
   return stockData ? (
     <div
-      id={`stock_${props.symbol}`}
+      id={`stock_${symbol}`}
       className={classNames({
         'card-wrapper': true,
         dragging: dragging.isDragging,
-        dragOver: props.dragOver,
-        dragOverBottom: props.dragOverBottom
+        dragOver: dragOver,
+        dragOverBottom: dragOverBottom
       })}
-      draggable={!props.isUnwatching}
-      onDragStart={onDragStart(props.symbol)}
+      draggable={!isUnwatching}
+      onDragStart={onDragStart(symbol)}
       onDragEnd={onDragEnd}
     >
       <div className="drop-target">
         <div className="card darkens default-background" draggable="false">
           <div className="card-top">
             <div className="details-container">
               <div className="name">{stockData.name}</div>
-              <div className="symbol">{props.symbol}</div>
+              <div className="symbol">{symbol}</div>
             </div>
             <div className="icons">
-              <Components.NewsShortcut
+              <div
```

Comment:
I think this might have regressed as well. It looks like the changes here are a move back to how it was before you introduced the shortcut components

Reply:
Yeah, thanks for spotting this, should be fixed now.